gltf-utils/src/lib.rs

```diff
@@ -49,7 +32,7 @@ pub trait Source: fmt::Debug {
 }
 
 /// Extra methods for working with `gltf::Primitive`.
-pub trait PrimitiveIterators<'a> {
+pub trait PrimitiveIterators<'a>: private::Sealed {
```

Comment:
What does this do exactly? I imagine it prevents further implementations but I'm not sure how.

Reply:
By not forwarding Sealed trait from mod "private" to public.  It will work as long as trait itself is marked pub and there are no plans to change visibility rules that would affect this.

Reply:
Okay, that makes sense.